freemarker/src/test/java/org/jdbi/v3/freemarker/BindBeanListTest.java

```diff
@@ -40,11 +39,11 @@
     private static List<Something> expectedSomethings;
 
     @ClassRule
-    public static final H2DatabaseRule dbRule = new H2DatabaseRule();
+    public static final H2DatabaseRule DB_RULE = new H2DatabaseRule();
```

Comment:
This would probably be better to change to an instance rule instead of a class rule.